static/change-root-profile.sh

```diff
@@ -44,9 +44,10 @@ ROOT-PROFILE
 # Add Aliases
 {
 echo "alias nextcloud_occ='sudo -u www-data php $NCPATH/occ'"
-echo "alias run_update_nextcloud='bash $SCRIPTS/update.sh'"
+echo "alias update_nextcloud='bash $SCRIPTS/update.sh'"
```

Comment:
I know we had that discussion already but I think that makes sense (is also only applicable to new users and hence doesn't destroy anything)

Reply:
Do you change the FAQ and DOCSas well? :)

Reply:
Yes, would do that. Could you send me the links to the related sites? 

Am not really experienced with the docs.

Reply:
BTW: was was the reason to not host the docu on github?
would probably make the collaboration on docs a bit easier....

Reply:
I've reverted the change because it doesn't have anything to do with the PR.
have opened #1233 for that.